packages/env/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### Breaking Change
+
+-   Removed the second shell from the execution of `run` commands. This removes the need for double-escaping and fixes problems with undocumented evaluation of escaped shell tokens before they are passed to the container. As a consequence of this change, quoted strings will now be treated as single arguments instead of separate ones. For example, `npx wp-env run cli "wp help"` will attempt to run `"wp help"` on the container instead of `wp` with `help` as an argument. If you are using commands with excess escaping or quotes you will need to review them and ensure they are compatible with the update. 
+
+### Enhancement
+
+-   Support using double dashes in `wp-env run ...` to pass arguments that would otherwise be consumed by `wp-env`. For example, while normally `--help` would provide the `wp-env` help text, if you use `npx wp-env run cli php -- --help` you will see the PHP help text.
```

Comment:
The thing that gets me about this change is that I have always hated the `-- --help` syntax from npm :/ 

I personally prefer `npx wp-env run cli 'php --help'` over `npx wp-env run cli php -- --help`

Reply:
Unfortunately, this is just a consequence of using `yargs`. It eats any options that aren't defined unless you use a double-dash to tell it to stop parsing everything after it.